src/languageServerInstaller.ts

```diff
@@ -70,7 +68,7 @@ export class LanguageServerInstaller {
       // Most of the time, getLsVersion will produce "ENOENT: no such file or directory"
```

Comment:
```suggestion
      // Most of the time, getLsVersion will produce "FileSystemError: FileNotFound"
```
I think `workspace.fs` is throwing `FileSystemError`s: https://code.visualstudio.com/api/references/vscode-api#FileSystemError